build.sh

```diff
@@ -25,226 +26,178 @@ fi
 set -eo pipefail
 
 exit_error() {
-    echo "$*" 1>&2
-    exit 1;
+	echo "$*" 1>&2
+	exit 1;
 }
 
 usage() {
-    cat <<-EOF
-	Usage: $(basename "${BASH_SOURCE[0]}") [-hu] [-k /path/to/authorized_keys] 
-	
+	cat <<-EOF
+	Usage: $(basename "${BASH_SOURCE[0]}") [-hu] [-k /path/to/authorized_keys]
+
 	-h Show help
 	-u Build with upload to remote host
-	-k /path/to/authorized_keys (can safely ignore, another opportunity to copy in SSH keys on image boot!)
+	-k /path/to/authorized_keys (can safely ignore, another opportunity to copy
+	   in SSH keys on image boot!)
 	EOF
 }
 
 # we must be on freebsd
 what_os_am_i="$(uname)"
-if [ "${what_os_am_i}" != "FreeBSD" ]; then
-    exit_error "Please run on FreeBSD only"
+if [ "$what_os_am_i" != "FreeBSD" ]; then
+	exit_error "Please run on FreeBSD only"
 fi
 
 UPLOAD="NO"
 # get command line flags
 while getopts huk: flag
 do
-    case "${flag}" in
-        h) 
-           usage
-           exit 0
-           ;;
-        u) 
-           UPLOAD="YES"
-           ;;
-        k) 
-           AUTHKEYFILE="${OPTARG}"
-           ;;
-        *) 
-           exit_error "$(usage)"
-           ;;
-    esac
+	case "$flag" in
+	h)
+		usage
+		exit 0
+		;;
+	u)
+		UPLOAD="YES"
+		;;
+	k)
+		AUTHKEYFILE="$(realpath "$OPTARG")"
+		;;
+	*)
+		exit_error "$(usage)"
+		;;
+	esac
 done
 shift "$((OPTIND-1))"
 
-# Set default values if not set
-# XXX: this might be unnecessary
-if [ -z "${AUTHKEYFILE}" ]; then
-    touch authorized_keys_in
-    AUTHKEYFILE="authorized_keys_in"
-fi
-
 # General Variables
 BASEDIR="$PWD"
 CDMOUNT="cd-rom"
-CHECKMOUNTCD1="$(mount | { grep "${CDMOUNT}" || :; } | awk '{print $1}')"
+CHECKMOUNTCD1="$(mount | { grep "$CDMOUNT" || :; } | awk '{print $1}')"
 FREEBSDISOSRC="https://download.freebsd.org/releases/amd64/amd64/ISO-IMAGES/13.1/FreeBSD-13.1-RELEASE-amd64-dvd1.iso"
 FREEBSDISOFILE="FreeBSD-13.1-RELEASE-amd64-dvd1.iso"
 MFSBSDDIR="mfsbsd"
 MYRELEASE="13.1-RELEASE"
 MYARCH="amd64"
-OUTIMG="mfsbsd-${MYRELEASE}-${MYARCH}.img"    # not in use
-OUTISO="mfsbsd-${MYRELEASE}-${MYARCH}.iso"    # in use
-MYBASE="${BASEDIR}/${CDMOUNT}/usr/freebsd-dist"
-MYCUSTOMDIR="${BASEDIR}/customfiles"
+OUTIMG="mfsbsd-$MYRELEASE-$MYARCH.img"    # not in use
```

Comment:
I removed quite a lot of unnecessary curly braces to safe a mit of space and visual noise